EnglishFilter/sections/antiadblock.txt

```diff
@@ -432,6 +432,8 @@ telva.com,gazettetimes.com,missoulian.com,kenoshanews.com,cumberlink.com,stltoda
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/58002
 hinjakuhonyaku.com#%#//scriptlet("abort-on-property-write", "ai_front")
+! https://github.com/AdguardTeam/AdguardFilters/issues/58232
+gsmfirmware.net#%#//scriptlet("abort-current-inline-script", "document.createElement", "ignielAdBlock")
```

Comment:
You can solve it without scriptlet. Use these rules - exception will work for content blockers too, so you don't to add rule in cb section.
```
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=gsmfirmware.net
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,redirect=googlesyndication-adsbygoogle,domain=gsmfirmware.net,important
```